snuba/admin/views.py

```diff
@@ -395,20 +398,26 @@ def clickhouse_system_query() -> Response:
         port = req["port"]
         storage = req["storage"]
         raw_sql = req["sql"]
+        sudo_mode = req.get("sudo", False)
     except KeyError:
         return make_response(jsonify({"error": "Invalid request"}), 400)
 
     try:
-        result = run_system_query_on_host_with_sql(host, port, storage, raw_sql)
+        result = run_system_query_on_host_with_sql(
+            host, port, storage, raw_sql, sudo_mode, g.user
```

Comment:
I am having trouble understanding what `g.user` means, can we rename it to something more readable?

Reply:
`g` in flask is the special variable provided by the framework to store request scoped data. 
So I cannot rename it. It is provided by the framework. https://flask.palletsprojects.com/en/2.3.x/api/#flask.g

Reply:
Thanks for the link. I didn't know it existed.